client/blocks/signup-form/index.jsx

```diff
@@ -1067,17 +1067,11 @@ function TrackRender( { children, eventName } ) {
 export default connect(
 	( state, ownProps ) => {
 		const isDisplayUsernamePropSet = ownProps.hasOwnProperty( 'displayUsernameInput' );
-		const eligibleFlowsForRemoveUsernameTest = [
-			'onboarding',
-			'personal',
-			'premium',
-			'business',
-			'ecommerce',
-		];
+		const eligibleFlowsForRemoveUsernameTest = [ 'personal', 'premium', 'business', 'ecommerce' ];
 		let displayUsernameInput = true;
 
 		if ( eligibleFlowsForRemoveUsernameTest.includes( ownProps.flowName ) ) {
-			abtest( 'usernameAATest' );
+			'control' === abtest( 'removeUsernameInSignup' );
```

Comment:
```suggestion
			displayUsernameInput = 'control' === abtest( 'removeUsernameInSignup' );
```
You probably meant this? :)

Reply:
Ah yes, I forgot to push that change, it was in my local commit. Just pushed, thanks for catching that.

Reply:
The commit [7ed2a28](https://github.com/Automattic/wp-calypso/pull/46038/commits/7ed2a287b3b901115aa3828b3754fece5c9a7bbd) also removes the A/A test.